test/integration/proposal-checks/IntegrationTestBorrowSDAICollateral.sol

```diff
@@ -0,0 +1,461 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.13;
+
+import "@forge-std/Test.sol";
+
+import {LendingTerm} from "@src/loan/LendingTerm.sol";
+import {PostProposalCheckFixture} from "@test/integration/proposal-checks/PostProposalCheckFixture.sol";
+
+contract IntegrationTestBorrowSDAICollateral is PostProposalCheckFixture {
+    function testTermParamSetup() public {
+        assertEq(term.collateralToken(), address(sdai));
+        {
+            LendingTerm.LendingTermParams memory params = term.getParameters();
+            assertEq(params.collateralToken, address(sdai));
+            assertEq(params.openingFee, 0);
+            assertEq(params.interestRate, SDAI_RATE);
+            assertEq(params.minPartialRepayPercent, 0);
+            assertEq(params.maxDelayBetweenPartialRepay, 0);
+            assertEq(params.maxDebtPerCollateralToken, MAX_SDAI_CREDIT_RATIO);
+        }
+        {
+            LendingTerm.LendingTermReferences memory params = term
+                .getReferences();
+
+            assertEq(params.profitManager, addresses.mainnet("PROFIT_MANAGER"));
+            assertEq(params.guildToken, address(guild));
+            assertEq(params.auctionHouse, addresses.mainnet("AUCTION_HOUSE"));
+            assertEq(params.creditMinter, address(rateLimitedCreditMinter));
+            assertEq(params.creditToken, address(credit));
+        }
+    }
```

Comment:
You can do this in Proposal_0's validate() step

Then, in this integration test suite, define state variables : 
```
string private constant TERM_NAME = "TERM_SDAI_1";
address private collateralToken;

function setUp() public {
  super.setUp();

  term = LendingTerm(addresses.mainnet(TERM_NAME));
  collateralToken = term.getParameters().collateralToken;
}
```

Then in `testSupplyCollateralUserOne` :
```
deal(collateralToken, userOne, supplyAmount);
```
etc, everywhere you're using the 'sdai' constant

This will allow this test to be more generic and easily updated (just have to change the TERM_NAME constant) if the set of live lending terms change.